configureBuild.sh

```diff
@@ -281,6 +281,9 @@ function setMakeArgs() {
     "darwin") BUILD_CONFIG[MAKE_ARGS_FOR_ANY_PLATFORM]=${BUILD_CONFIG[MAKE_ARGS_FOR_ANY_PLATFORM]:-"product-images mac-legacy-jre-bundle"} ;;
     *) BUILD_CONFIG[MAKE_ARGS_FOR_ANY_PLATFORM]=${BUILD_CONFIG[MAKE_ARGS_FOR_ANY_PLATFORM]:-"product-images legacy-jre-image"} ;;
     esac
+    if [ "${BUILD_CONFIG[MAKE_EXPLODED]}" == "true" ]; then
+      BUILD_CONFIG[MAKE_ARGS_FOR_ANY_PLATFORM]=""
```

Comment:
Could this be subsequently overridden?  We might need to do this later or stick warning label on it.

Reply:
this is the latest place it can be set

Reply:
A comment would be useful why `MAKE_ARGS_FOR_ANY_PLATFORM` is set to `""` and there might need some sort of guidance for future editors that they need to ensure that `MAKE_ARGS_FOR_ANY_PLATFORM` stays empty.